packages/frontend/src/views/CalendarView/CalendarView.module.css

```diff
@@ -0,0 +1,51 @@
+.sidebarHeadings {
+  font-size: var(--flocker-font-size-medium);
+  font-weight: var(--flocker-font-weight-light);
+  margin-top: 16px;
+}
+
+.timeMatcher {
+  width: 60vw;
+  height: 70vh;
+}
+
+h2 {
+  font-size: var(--flocker-font-size-body);
+  font-weight: var(--flocker-font-weight-light);
+  padding: 16px 0 8px 0;
+}
+
+h3 {
+  font-size: var(--flocker-font-size-small);
+  font-weight: var(--flocker-font-weight-light);
+  padding: 0 10px 0 10px;
```

Comment:
FYI, if you have the top+bottom padding the same, and the left+right padding the same, you can do `padding: 0 10px`. Don't need to change, just letting you know 😄

Reply:
Will change hehe that looks better